examples/ShapeEditing.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
```

Comment:
Could we call this example ShapeEditor.html/.js so it matches with the feature name?